src/appendices/phenotype.md

```diff
@@ -0,0 +1,250 @@
+
+When a sessions file is in use, you MUST NOT provide participant-level sessions files which would otherwise use the inheritance principle. If a sessions file is provided, then it MUST begin with a `participant_id` column followed immediately by a `session_id` column. The data dictionary JSON’s `session_id` field MUST include `Levels` with the description of each possible `session_id`.
+
+### 4. Sessions age
+
+It is RECOMMENDED to use the `age` column to record participant age at every session. This reduces data duplication across tabular data files. The `Units` of `age` do not have to be years so long as the units of the age are written in `sessions.json`.
+
+### 5. Sessions `acq_time`
+
+Whenever possible, it is RECOMMENDED to also collect acquisition time for tabular phenotypic data and store the time of acquisition[^2] of each row inside a column named `acq_time` in the sessions file.
+
+When needed to preserve participant privacy, you SHOULD record relative acquisition times with respect to the earliest session. Relative session acquisition times MAY be listed as durations from the earliest session (baseline) in days, months, or years using the `acq_time` column.
+
+### 6. Demographics file
+
+Some studies collect demographics into their own tabular phenotypic data file already. In these cases, it is RECOMMENDED to house this data in the `phenotype/` directory as a TSV called `demographics.tsv` and its corresponding data dictionary JSON called `demographics.json`.
+
+### 7. Summary
+
+This appendix described the guidelines for the best tabular phenotypic data. A short summary table here describes when to use which files.
+
+| File                           | Single session data | Multiple session data |
+| :----------------------------- | :------------------ | :-------------------- |
+| Participants                   | RECOMMENDED         | RECOMMENDED           |
+| Phenotypic and assessment data | RECOMMENDED         | RECOMMENDED           |
+| Sessions                       | OPTIONAL            | REQUIRED              |
+| Demographics                   | OPTIONAL            | RECOMMENDED           |
+
+## Examples
+
+What follows are a few common use case examples for tabular phenotypic files.
+
+### 1 participant session with both non-tabular and tabular phenotypic data
+
+File tree
+
+```text
+phenotype/
+    tool.json
+    tool.tsv
+sub-01/anat/
+    sub-01_T1w.json
+    sub-01_T1w.nii.gz
+```
+
+Contents of `phenotype/tool.tsv`
+
+```text
+participant_id	measurement_1	measurement_2
+sub-01	value1	value2
+```
+
+### 1 participant with 2 sessions, where 1 session is only tabular phenotype and the other is only imaging
+
+With only two sessions like this you might want to make it look like one session, but it is more correct to have the sessions separated and have `session_id` `Levels` noted in the `sessions.json`, especially if the sessions were collected days, weeks, or months apart.
+
```

Comment:
```suggestion
With only one imaging and one phenotypic session each in this example you might want to make merge both imaging and phenotypic data under one session. But it is more correct to have separate sessions for the imaging and phenotypic data, especially if the sessions were collected days, weeks, or months apart. You can denote both sessions and their acquisition time and in the sessions.tsv file have `session_id` `Levels` noted in the `sessions.json` sidecar.
```

Reply:
I am an easily distracted reader, so if you spell things out for me, I can follow better.